src/sentry/static/sentry/less/components/button.less

```diff
@@ -0,0 +1,87 @@
+@import url("../palette.less");
```

Comment:
This was from a previous PR from @benvinegar and @ckj, I believe their intention was to isolate this component from our legacy `btn`s. This PR only changes the 2 buttons in the header of org-home in order to minimize scope of PR, so we'll want to migrate to this new component over time.

 `getsentry` will be able to use this if we export to global ns.